sbt-dumpschema/src/main/scala/org/virtuslab/akkasaferserializer/DumpSchema.scala

```diff
@@ -0,0 +1,18 @@
+package org.virtuslab.akkasaferserializer
+
+import org.virtuslab.akkasaferserializer.DumpSchemaPlugin.autoImport.{dumpSchemaFilename, dumpSchemaOutputPath}
+import sbt.Keys.{packageOptions, streams, test}
+import sbt.{Def, Task, TaskKey}
+
+import java.io.File
+
+object DumpSchema {
+  def apply(filename: File): File = {
+    filename
+  }
+
+  def assemblyTask(key: TaskKey[File]): Def.Initialize[Task[File]] =
+    Def.task {
+      DumpSchema((dumpSchemaOutputPath in key).value)
```

Comment:
Actually! One non-obvious thing that's pretty important (at least while we're still using Jackson in Hydra...) is that **class annotations** (like `@JsonTypeInfo` etc.) are included in the dump as well